src/fixto.js

```diff
@@ -326,7 +327,7 @@ var fixto = (function ($, window, document) {
                     }
                 }
             }
-            return top;
+            return top + this.options.top;
```

Comment:
Yeah, all 100 percent of usages of this method were along with this option. I'll fix the description then